data/mods/Aftershock/README.md

```diff
@@ -39,4 +40,4 @@ Contains json data for vehicles, vehicle parts, and vehicle spawn groups.
 
 ### npcs
 
-Contains json data for npcs, factions, and design document for any new factions, specific new faction PrepPhyle.  Bionic Preppers living simple farming life with bionic upgrades, prefer to upgrade animals instead of relying on robots.
+Contains json data for npcs, factions, and design document for any new factions, specific new factions PrepPhyle and Whately Clan.  The PrepPhyle are a group of frontier squatters taking advantage of decaying colony infrastructure to set up an independent colonoy separate from Corporate and Earthgov influences.  The Whately Clan are a possibly exiled branch of a corporate executive family.  They are mad, bad, and dangerous to know.
```

Comment:
Prepnet, or PrepPhyle? Both are mentioned in file

Reply:
It's PrepNet Phyle.  So both are shortened versions.